### PR TITLE
Rule 323: Automaticvote

### PR DESCRIPTION
**Pull request by MrHug**
Voting for your own proposals seems a bit redundant...
For the sake of "getting some voting started" explicit voting is still allowed, overruling the implicit vote.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,12 @@ gains 1 point.
 
 **315** Each eligible voter always has exactly one vote.
 
+**323** It may be assumed that the proposer of a rule-change is always in favour of this rule-change and thus has
+cast an implicit vote in favour of the rule-change. 
+An explicit vote through the procedures outlined in the rules may replace this implicit vote in favour.
+If the implicit vote is not overwritten with an explicit vote, the implicit vote is always valid, no matter what other
+rules say about the validity of votes.
```
Comment (jdonkervliet) [inline README.md]: Remove these last two lines.

Comment (MrHug) [inline README.md]: You mean the "If the implicit vote is not overwritten with an explicit vote, the implicit vote is always valid, no matter what other rules say about the validity of votes." part?

Why would you like this removed? The reason I included it is that this implicit vote is valid, even if the original PR does not contain Turkish/Japanese.

Comment (jdonkervliet) [inline README.md]: It says 'always valid' and 'no matter what ...'. This makes me very very wary.

Comment (MrHug) [inline README.md]: Hmm, I can imagine, but can you describe circumstances in which you would not want to back up your own PR without explicitly mentioning this?

**Comment by arthurbik** [comment]
If you do not have the right to vote, this allows you to cast a vote without actually voting.

**Comment by arthurbik** [comment]
"An explicit vote through the procedures outlined in the rules may replace this implicit vote in favour."
please change this to
"An explicit vote through the procedures outlined in the rules replaces this implicit vote in favour."

File: README.md (hunk 1 above)
Comment (jdonkervliet) [inline README.md]: I won't. Just rephrase.

Comment (MrHug) [inline README.md]: ... Alright then...

**Comment by MrHug** [comment]
Done, although the validity of this PR is still pending, awaiting the Verdict of #34.

**Comment by jdonkervliet** [comment]
Testing if @Jason-Statham is working.

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by jdonkervliet** [comment]
:+1:

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by MrHug** [comment]
:+1:

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by arthurbik** [comment]
:+1:

**Comment by Jason-Statham** [comment]
jdonkervliet voted in favor.
MrHug voted in favor.
arthurbik voted in favor.
That is 3 out of 4 players. This PR should be merged. I cannot do that yet.
happy

**Comment by MrHug** [comment]
Although the majority has been reached, this PR is on hold until a verdict is given in #34 as that might influence the validity of this PR.

Even if it were not the topic of discussion a formal agreement on whether or not merging is allowed during a Trial is also lacking.

**Comment by MrHug** [comment]
With 3 votes in favor, among 4 players, a simple majority has been achieved, so by rule 203, the suggested rule-change is adopted. As prescribed by rule 314, I will merge this pull request.

**Pull request merged**